CalSmokeApp/bin/make/ipa-cal.sh

```diff
@@ -133,7 +133,7 @@ fi
 cat >"${XTC_DIR}/Gemfile" <<EOF
 source "https://rubygems.org"
 
-gem "calabash-cucumber", "2.99.0"
```

Comment:
The same as above: why do we need to do this?

Reply:
In this case I see this error:
`Could not find gem 'calabash-cucumber (= 2.99.0)' in any of the gem sources`

Reply:
The latest version of `calabash-cucumber` gem is 0.21.x